ext/wiiu/elf2rpl/Makefile

```diff
@@ -0,0 +1,27 @@
+TARGET   := elf2rpl
```

Comment:
Can we use a submodule?  Is there a git repo for this?

-[Unknown]

Reply:
there is yeah : https://github.com/decaf-emu/wut/tree/master/tools/elf2rpl
I did modify the code a bit though and I'm not sure if they want it upstreamed.

I could in theory also move the whole ext/wiiu folder to a different repo and make it a submodule instead, if that's preferred.

Reply:
Ah, if it's customized, that's different.  I just saw it was 20K SLOC and figured it'd probably get updates down the line.

-[Unknown]

Reply:
ah I assumed you were just referring to elf2rpl.
the ext/wiiu folder is just a small sdk I've been building up while working on different projects, so any updates it will see in the near future would be the ones done here anyway.
I do have a repo set up for this but I didn't bother updating it for a long time xD